zhaquirks/tuya/ts0205.py

```diff
@@ -0,0 +1,118 @@
+"""Tuya TS0205 smoke detector."""
+import logging
+
+from zigpy.profiles import zha
+from zigpy.quirks import CustomDevice
+import zigpy.types as t
+from zigpy.zcl.clusters.general import (
+    Basic,
+    Groups,
+    Identify,
+    OnOff,
+    Ota,
+    PowerConfiguration,
+    Scenes,
+    Time,
+)
+from zigpy.zcl.clusters.lightlink import LightLink
+from zigpy.zcl.clusters.security import IasZone
+
+from zhaquirks import LocalDataCluster
+from zhaquirks.const import (
+    DEVICE_TYPE,
+    ENDPOINTS,
+    INPUT_CLUSTERS,
+    MODELS_INFO,
+    OUTPUT_CLUSTERS,
+    PROFILE_ID,
+)
+from zhaquirks.tuya import TuyaManufClusterAttributes
+
+_LOGGER = logging.getLogger(__name__)
+
+TUYA_SMOKE_DETECTED_ATTR = 0x0401  # [0]/[1] [Detected]/[Clear]!
+
+
+class TuyaSmokeDetectorCluster(TuyaManufClusterAttributes):
+    """Manufacturer Specific Cluster of the TS0205 smoke detector."""
+
+    attributes = {
+        TUYA_SMOKE_DETECTED_ATTR: ("smoke_detected", t.uint8_t, True),
+    }
```

Comment:
Can we use the new `AttributeDefs` style here? As an example, see:
https://github.com/zigpy/zha-device-handlers/blob/915ff9b41e7393e1f2eaa449bd6a09267fa84277/zhaquirks/legrand/cable_outlet.py#L51-L68

You can also remove `TUYA_SMOKE_DETECTED_ATTR` then and if it needs to be access anywhere else (except the attribute definition, where it should just be "hard-coded"), `TuyaSmokeDetectorCluster.AttributeDefs.smoke_detected.id` can be used.

Reply:
Changed.